divi/src/CoinMinter.cpp

```diff
@@ -13,7 +13,7 @@
 #include <SuperblockHelpers.h>
 
 #include <BlockFactory.h>
-
+#include "main.h"
 
```

Comment:
As above

Reply:
it depends of bool ProcessNewBlock(CValidationState& state, CNode* pfrom, CBlock* pblock, CDiskBlockPos* dbp) which is   part of main.cpp , it will move in the next PR